gravitee-apim-rest-api/gravitee-apim-rest-api-management/gravitee-apim-rest-api-management-rest/src/main/java/io/gravitee/rest/api/management/rest/resource/ApiSubscribersResource.java

```diff
@@ -78,11 +86,11 @@ public class ApiSubscribersResource extends AbstractResource {
         description = "Paged result of API subscribers",
         content = @Content(
             mediaType = MediaType.APPLICATION_JSON,
-            array = @ArraySchema(schema = @Schema(implementation = ApplicationEntity.class))
+            array = @ArraySchema(schema = @Schema(implementation = ApplicationListItem.class))
         )
     )
     @ApiResponse(responseCode = "500", description = "Internal server error")
-    public Collection<ApplicationEntity> getApiSubscribers() {
+    public Collection<ApplicationListItem> getApiSubscribers() {
```

Comment:
Need to document this as a breaking change as the return isn't the same anymore? Explaining we've done it for performances reasons

Reply:
Yes, a PR in the gravitee-docs repo I guess ?

FYI: there is only 3 differences in ApplicationListItem:
 - pictureURL (new in ApplicationListItem)
 - backgroundURL (new in ApplicationListItem)
 - disableMembershipNotifications (missing in ApplicationListItem)